docs/source/scilla-in-depth.rst

```diff
@@ -2648,20 +2898,94 @@ An example ``init.json``:
    ]
 
 
-Chain Invocation Behaviour
-**************************
+Chain Calls
+***********
+.. _Chaincalls:
+
+When a user invokes a transition on a contract by sending a message
+with the contract's address as the recipient, then that transition may
+then send one or more messages onwards, possibly invoking other
+transitions on other contracts. The resulting collection of messages,
+fund transfers, transition invocations, and contract state changes are
+referred to as a `transaction`.
+
+A transition that sends a message invoking another transition
+(typically on another contract) is referred to as a `chain call`.
```

Comment:
```suggestion
(typically on another contract) is referred to as a *chain call*.
```

Reply:
Fixed, and I've changed the emphasis of "transaction" in the previous paragraph as well. It's not the first place we use the word "transaction", but that paragraph contains the definition.